pkg/v6/configmap/desired.go

```diff
@@ -38,7 +39,7 @@ type IngressControllerGlobal struct {
 }
 
 type IngressControllerGlobalController struct {
-	Replicas         int  `json:"replicas"`
+	TempReplicas     int  `json:"tempReplicas"`
```

Comment:
Renamed to tempReplicas to make the name clearer. See https://github.com/giantswarm/kubernetes-nginx-ingress-controller/pull/51 for the updated chart.